src/AppInstallerCommonCore/AppInstallerStrings.cpp

```diff
@@ -268,4 +268,9 @@ namespace AppInstaller::Utility
 
         return result;
     }
+
+    std::string ToString(const std::string_view key)

```

Comment:
>ToString [](start = 16, length = 8)

this is 2 chars less than std::string, but a bit more confusing as one doesn't know exactly what is going to happen without reading it.

Please remove and just use std::string()